ui/src/components/auth/helpers/form.types.ts

```diff
@@ -0,0 +1,22 @@
+type LoginFormFields = {
+  email: string;
+  password: string;
+};
+
+type RegisterFormFields = {
+  fullName: string;
+  email: string;
+  password: string;
+  confirmPassword: string;
+};
+
+type RecoverPasswordFields = {
+  email: string;
+};
+
+type ResetPasswordFields = {
```

Comment:
can use your `Omit` thingy here? and also for all of these schema, since `RegisterFormFields` has everything